x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useState, useMemo } from 'react';
+import { EuiDataGrid, EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
+import { useKibana } from '../../../common/lib/kibana';
+import { QueryBar, useQueryBar, Provider, alertsPageStateContainer } from './query_bar';
+import { getVisibleAlertConsumers } from './lib/get_visible_alert_consumers';
+import { useFetchDataViews, useSorting, usePagination } from './hooks';
+import { AlertsTableProps } from '../../../types';
+
+const AlertsTableComponent: React.FunctionComponent<AlertsTableProps> = (
+  props: AlertsTableProps
```

Comment:
accept and use a `data-test-subj` here for easier unit and e2e testing.

Reply:
will do